packages/@glimmer/runtime/lib/dom/normalize.ts

```diff
@@ -43,6 +44,18 @@ export function isEmpty(value: unknown): boolean {
   return value === null || value === undefined || typeof (value as Dict).toString !== 'function';
 }
 
+let isHTML: ((value: unknown) => boolean) | undefined;
+if (typeof window !== 'undefined') {
+  let trustedTypes = (window as unknown as TrustedTypesWindow).trustedTypes;
```

Comment:
this is a good way to progressively allow the feature for browsers that have it and not break in browsers that don't